<launch>

<rosparam command="load" file="$(find vi_ekf)/params/gains.yaml" ns="vi_ekf_node"/>
<rosparam command="load" file="$(find vi_ekf)/params/ocam.yaml" ns="vi_ekf_node"/>

<!-- <node name="vi_ekf_node" type="vi_ekf_rosbag" pkg="vi_ekf" output="screen" args="/home/superjax/stationary_ocam.bag -r"/> -->
<node name="vi_ekf_node" type="vi_ekf_rosbag" pkg="vi_ekf" output="screen" args="/home/superjax/ocam_roll.bag -r"/>

<node name="tracked_view" type="image_view" pkg="image_view">
	<remap from="image" to="tracked"/>
</node>

<!--<node name="record" type="record" pkg="rosbag" args="-O /home/superjax/rosbag/vi_ekf_results /odom /imu /truth/pose" output="screen"/>-->


</launch>
